pymchelper/readers/fluka.py

```diff
@@ -268,3 +284,70 @@ def read_data(self, estimator, nscale=1):
 
         estimator.file_format = 'fluka_binary'
         return True
+
+
+def get_particle_from_db(particle_id: int) -> Optional[Particle]:
+    """Get particle from Flair database by its id"""
+    try:
+        Particle.makeLists()
+        particle = Particle.get(particle_id)
+        return particle
+    except KeyError:
+        return None
+
+
+class UsrbinScoring:
+    """Scoring names for USRBIN estimator"""
+
+    _deposition_scorings = [
+        'ENERGY', 'EM-ENRGY', 'DOSE', 'UNB-ENER', 'UNB-EMEN', 'NIEL-DEP', 'DPA-SCO', 'DOSE-EM', 'DOSEQLET', 'RES-NIEL'
+    ]
+    _fission_density_scorings = ['FISSIONS', 'HE-FISS', 'LE-FISS']
+    _neutron_balance_desnity_scorings = ['NEU-BALA']
+    _density_of_momentum_scorings = ['X-MOMENT', 'Y-MOMENT', 'Z-MOMENT']
+    _activity_scorings = ['ACTIVITY', 'ACTOMASS']
+    _dose_equivalent_scorings = ['DOSE-EQ']
+    _fluence_weighted_bdf_scorings = ['SI1MEVNE']
+    _he_tn_fluence_scorings = ['HEHAD-EQ', 'THNEU-EQ']
+    _net_charge_scorings = ['NET-CHRG']
+
+    @classmethod
+    def get_scoring_and_unit(cls, scoring_or_particle: str) -> Tuple[str, str]:
```

Comment:
Can you make it a simple method which takes `generalized_particle` as the input and simply returns units ?
I see no reason to make a class or return the input argument